config/serverless.oblt.yml

```diff
@@ -12,111 +12,21 @@ xpack.legacy_uptime.enabled: false
 
 ## Fine-tune the observability solution feature privileges. Also, refer to `serverless.yml` for the project-agnostic overrides.
 xpack.features.overrides:
-  ### Applications feature privileges are fine-tuned to grant access to Logs, and Observability apps.
-  apm:
-    ### By default, this feature named as `APM and User Experience`, but should be renamed to `Applications`.
-    name: "Applications"
-    privileges:
-      # Infrastructure's `All` feature privilege should implicitly grant `All` access to Logs and Observability apps.
-      all.composedOf:
-        - feature: "logs"
-          privileges: [ "all" ]
-        - feature: "observability"
-          privileges: [ "all" ]
-      # Infrastructure's `Read` feature privilege should implicitly grant `Read` access to Logs and Observability apps.
-      read.composedOf:
-        - feature: "logs"
-          privileges: [ "read" ]
-        - feature: "observability"
-          privileges: [ "read" ]
+  ### By default, this feature named as `APM and User Experience`, but should be renamed to `Applications`.
```

Comment:
Remove all extra privileges included in `composedOf`.

- Logs: We decided to continue including Logs as a top level privilege
- Observability: We've removed the privilege entirely. This is possible because there aren't any roles that were ever created with the 'observability' privilege. Rather, since we've continued having editor, viewer, and admin roles since serverless has been GA, those roles specify essentially a wildcare privilege that includes all Kibana privilege, not a specific enumerated list of privileges.